### PR TITLE
Adds basic frontend for manually tracked balances

### LINKED ISSUE #807: Allow tracking without "account"
## Abstract

Allow tracking of funds without specifying an account. E.g. some users might have physical coins where the public address is unknown and would only be revealed once the hologram is pulled off

## Motivation

Allow tracking of funds without account

## Specification

Above.

### PR DESCRIPTION
**Pull request by kelsos**
Work on the frontend part

Closes #807

### CODE REVIEW

**Comment by LefterisJP** [comment]
Is this based on https://github.com/rotki/rotki/pull/847? IF not that's fine but to test it nicely we would probably need to rebase on develop once #847 is merged.

Won't be too long now. It's almost ready. Adding the endpoint to query all assets.

**Comment by kelsos** [comment]
@LefterisJP No it's based on develop and I will rebase.

**Comment by LefterisJP** [comment]
@kelsos great, that's gonna be easier. Just rebase once I merge that PR.

File: electron-app/src/services/types-common.ts [hunk 1]
```diff
@@ -1 +1,25 @@
...
+
+export enum Location {
+  EXTERNAL = 'external',
+  KRAKEN = 'kraken',
+  POLONIEX = 'poloniex',
+  BITTREX = 'bittrex',
+  BINANCE = 'binance',
+  BITMEX = 'bitmex',
+  COINBASE = 'coinbase',
+  TOTAL = 'total',
```
Comment (LefterisJP) [inline electron-app/src/services/types-common.ts:11]: Can you remove this option (`TOTAL`)? It makes no sense in the context of adding locations. It's only used by the backend to show the total balances.

File: electron-app/src/services/types-common.ts [hunk 2]
```diff
@@ -1 +1,25 @@
...
+  BANKS = 'banks',
+  BLOCKCHAIN = 'blockchain',
+  COINBASEPRO = 'coinbasepro',
+  GEMINI = 'gemini'
+}
+
+export interface SupportedAsset {
+  readonly active: boolean;
```
Comment (LefterisJP) [inline electron-app/src/services/types-common.ts:19]: The `active`, `ended`, `started` attributes may not exist in an asset. Is this taken into account?

File: electron-app/src/components/inputs/AssetSelect.vue [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    :menu-props="{ closeOnClick: true, closeOnContentClick: true }"
+    item-value="symbol"
+    @input="input"
+  >
+    <template #selection="{ item }">
+      <span>{{ item.symbol }}</span>
+    </template>
+    <template #item="{ item }">
+      <v-list-item-avatar>
+        <crypto-icon :symbol="item.symbol"></crypto-icon>
```
Comment (LefterisJP) [inline electron-app/src/components/inputs/AssetSelect.vue:20]: Okay so now here we may have a problem.

As I wrote in the docs the symbol may not be unique. BUT it is indeed what the assets is known as. Unique is only going to be the key of the mapping.

Check for example the 3 different assets with the symbol `ACC` at `all_assets.json`:

https://github.com/rotki/rotki/blob/14b895bab0fad654d7abbacd2d1708255297e26e/rotkehlchen/data/all_assets.json#L94-L114

For now to guarantee uniqueness we make sure the keys are following the `SYMBOL-N` if multiple assets use a symbol.

The problem I see with the approach here is that the UI would show 3 different assets with `ACC` and the user can not know which is which.

One idea here is. Can we -- only for assets whose key contains `-N`, meaning multiple assets with the same symbol exist -- also add the name?

EDIT: Looking at the actual result of typing `ACC` into the search it seems even worse. Only 1 ACC is returned and the others are ignored.

2020-04-12-004332_1204x263_scrot

File: electron-app/src/components/accounts/ManualBalances.vue [hunk 4]
```diff
@@ -0,0 +1,34 @@
+<template>
```
Comment (LefterisJP) [inline electron-app/src/components/accounts/ManualBalances.vue:1]: Please also add some tests for all these

**Comment by LefterisJP** [comment]
Hmmm wtf .. never seen this error before in the tests.

```
=================================== FAILURES ===================================
_________________________ test_create_usage_analytics __________________________
    def test_create_usage_analytics():
        analytics = create_usage_analytics()
    
        assert 'system_os' in analytics
        assert 'system_release' in analytics
        assert 'system_version' in analytics
        assert analytics['rotki_version'] == get_system_spec()['rotkehlchen']
>       assert analytics['country'] != 'unknown'
E       AssertionError: assert 'unknown' != 'unknown'
rotkehlchen/tests/unit/test_usage_analytics.py:12: AssertionError
```

Must have meant that the ip of Travis momentarily did not map to a country

**Comment by LefterisJP** [comment]
@kelsos Looks fine now. Please make an issue for the rest of the tests and handle them soon. Next frontend task is coming soon too so keep it in mind :)

**Pull request merged**